mail_debrand/tests/test_mail_debrand.py

```diff
@@ -65,3 +65,19 @@ def test_plaintext_email(self):
         }
         # No exception expected
         MailMessage.create(email_values)
+
+    def test_body_intact(self):
+        """The message body should never be changed"""
+        MailMessage = self.env["mail.mail"]
+        original_body = """<p>And if I send odoo.example.com<br><br>And odoo.com<br><br>And <a target="_blank" rel="noreferrer noopener" href="https://odoo.com">https://odoo.com</a><br><br>And <a target="_blank" rel="noreferrer noopener" href="https://odoo.example.com">https://odoo.example.com</a></p>"""  # noqa: B950
```

Comment:
Can you please split this by lines (and the other one)?

Reply:
Done :+1: Thanks!